README.md

```diff
@@ -45,6 +45,10 @@
 * ![](http://ruby-gem-downloads-badge.herokuapp.com/reek?type=total)
 * ![](http://ruby-gem-downloads-badge.herokuapp.com/reek?label=downloads-current-version)
 
+## Reek 5 is out!
+
+TODO
```

Comment:
In the same vein of "let's not alienate our existing users that upgrade" we should make it prominent that there is a new, breaking release out there, at least for half a year or so.